features/facebookImport/src/context/facebook-context.jsx

```diff
@@ -1,17 +1,17 @@
 import React, { useEffect, useState } from "react";
 
 import i18n from "!silly-i18n";
-import { useHistory, useLocation } from "react-router-dom";
+import { useNavigate, useLocation } from "react-router-dom";
 
 import popUps from "../popUps";
 
 export const FacebookContext = React.createContext();
 
-function updatePodNavigation(pod, history, handleBack, location) {
+function updatePodNavigation(pod, navigate, handleBack, location) {
     pod.polyNav.actions = {
         back: () => handleBack(),
     };
-    history.length > 1 &&
+    navigate > 1 &&
```

Comment:
Gonna look into this one - should be a function how I understand it, not a number.